README.md

```diff
@@ -56,6 +56,10 @@ Example usage
 ted2zim --topics="augmented reality" --debug --name="augumented_reality" --format=mp4 --title="Augmented Reality" --description="TED videos in AR category" --creator="TED" --publisher="openzim" --output="output" --keep --low-quality
 ```
 
```

Comment:
Add two phrases, one before the "by-topic" example explaining that you can retrieve all talks of a given topic, one before your new example explaining that you can retrieve only selected talks by using their link